### DISCUSSION TITLE
Notification Service

### DISCUSSION THREAD

**Discussion by rnbokade** [supabase/supabase#6930] [Feature Requests]
Supabase already integrates SMS and Email service for authentication. Maybe same could be leveraged for Email and SMS notifications. This can futher be extended to web-push notifications and Apple Notifications.
I am not sure if FCM based notifications will be feasible especially when Supabase is being pitched as Firebase alternative. But atleast rest of the notification services can be implemented. 

#Additional context
In some regions like India, promotional messages like general notification have to be different sender id from general static template messages like that are to be sent for OTP. So additional configuration option Notification_SMS_Provider option which would default to be equal to what used for auth would need to be implemented

**Comment by kiwicopple** [comment]
Hey @rnbokade - this sounds like something we can do in the future, but not our immediate focus. I'll convert this one to a Discusssion so that others can contribute ideas here, and when we're ready we can incorporate all the feedback into our implementation

Thanks for the feature request!

**Comment by Olyno** [comment]
It could be interesting for Supabase to work with [Novu](https://github.com/novuhq/novu) (MIT license). Novu is an open-source notifications infrastructure to send SMS, E-mail, Slack and Push notifications. It also has an embeddable notification center for React with real-time updates, content management... This project looks promising.
